master/buildbot/newsfragments/custom_templates.bugfix

```diff
@@ -0,0 +1 @@
+custom_templates are now working again.
```

Comment:
Wondering if it's worth having 2.4.1 with this fix. I guess it's dealbreaker for some people isn't it?

Reply:
we had one report over IRC. not sure if it is worth urgent fix. Probably wait a bit more for more issues